packages/components/text/src/text.visualroute.jsx

```diff
@@ -100,7 +100,9 @@ export const component = () => (
       <Text.Body>Body text</Text.Body>
     </Spec>
     <Spec label="Body - bold">
-      <Text.Body isBold={true}>Body text bold</Text.Body>
+      <Text.Body isBold={true} fontWeight="bold">
```

Comment:
I think we should only use `fontWeight` prop in these tests.

Reply:
Good catch, I missed this in the first pass through. Thanks!